API.md

```diff
@@ -981,7 +981,7 @@ Platforms: iOS, Android
 
 ##### Overriding the metadata of a source
 
-Provide an optional `title`, `subtitle`  and/or `description` properties for the video. 
+Provide an optional `title`, `subtitle`, `customImageUri` and/or `description` properties for the video. 
```

Comment:
Is there any limitation for the image ?
I see the internal tvOS api need a png, but not clear if a png is mandatory in the API

Reply:
No the API supports the image formats supported by UIImage, which contains the classic ones like jpeg, png.
The list can be found here:

https://developer.apple.com/library/archive/documentation/2DDrawing/Conceptual/DrawingPrintingiOS/LoadingImages/LoadingImages.html